mkosi

```diff
@@ -1283,6 +1283,17 @@ def patch_file(filepath: str, line_rewriter: Callable[[str], str]) -> None:
     shutil.move(temp_new_filepath, filepath)
 
 
+def disable_pam_securetty(workspace: str,
+                          file_path: str = 'root/etc/pam.d/login') -> None:
+    patch_file(os.path.join(workspace, file_path), _rm_securetty)
+
+
+def _rm_securetty(line: str) -> str:
```

Comment:
Maybe make this function local to `disable_pam_securetty`?

Reply:
Done